tests/integration/test_association.py

```
@@ -365,9 +365,30 @@ def filter_debug_packages(type, cfg_packages_whitelist):
     return pkgs_expected_exclude
 
 
+def run_pub_client_publish(repo_ids, target):
+    cmd = ["pub", "publish", "--target", target]
+    for repo_id in repo_ids:
+        cmd.extend(["--repo", repo_id])
+    try:
+        ret = subprocess.check_output(cmd, stderr=subprocess.STDOUT).decode("utf-8")
+        if "CLOSED" not in ret:
+            print("The publish is not successful, so please check it first")
+            raise Exception(ret)
+        else:
+            print("The publish is successful")
+    except subprocess.CalledProcessError as ex:
```

you probably want to re-raise this exception, otherwise tests will continue with unpublished repos.

---

fixed